bob-apps/bin/ccg.rs

```diff
@@ -180,23 +181,21 @@ fn get_replicas_count(matches: &ArgMatches) -> AnyResult<usize> {
         .map_err(|err| anyhow!("get replicas count: {}", err))
 }
 
-fn get_vdisks_count(matches: &ArgMatches, nodes: &[ClusterNode]) -> AnyResult<usize> {
-    matches.value_of("vdisks_count").map_or_else(
-        || {
-            let res = get_pairs_count(nodes);
-            debug!("get vdisks count: OK [{}]", res);
-            Ok(res)
-        },
-        |s| {
-            s.parse()
-                .map_err(|err| anyhow!("get vdisks count: {}", err))
-        },
-    )
+fn get_vdisks_count(matches: &ArgMatches) -> AnyResult<usize> {
+    if let Some(s) = matches.value_of("vdisks_count") {
+        s.parse()
+            .map_err(|err| anyhow!("get vdisks count: {}", err))
+    } else if let Some(s) = matches.value_of("vdisks_per_disk") {
+        s.parse()
```

Comment:
This conflicts with the function name because it returns the number of virtual disks per disk (not the number of virtual disks)